docs/processes/quality-assurance/testing-and-dev.md

```diff
@@ -0,0 +1,200 @@
+# QA Process - Testing and Development
+
+[Testing and Development](#testing-and-development
+
+- [Feature Development: where to start?](#feature-development-–-where-do-i-start)
+
+- [Test-Driven Development](#test-driven-development)
+  - [Writing Appropriate Tests](#determining-appropriate-tests)
+  - [Writing a Failing Test Case](#writing-a-failing-test-case)
+  - [Writing Code to Pass a Test](#writing-code-to-pass-your-test)
+  - [Refactoring Code](#refactoring-code)
+- [Epics and User Stories](#epics-and-user-stories)
+  - [Epics](#epics)
+  - [User Stories](#user-stories)
+- [Testing Templates](#testing-templates)
+
+Current Areas under construction:
+
+- [Refactoring Intro](#tbc-refactoring-intro)
+- [Waiting epic Template link](#tbc-template-link)
+- [Testing tools expansion](#tbc-testing-tools)
+
+# Testing and Development
+
+Testing should be built into the software development process.
+
+Thoth-Tech advocates for a [Test Driven Development](#test-driven-development) approach.
+
+But you might be wondering, before jumping into coding and planning tests for a new feature, _where do you begin_?
+
+## Feature development – where do I start?
+
+- Create [epics](#epics) for each feature for planned development
+- Identify [user stories](#user-stories) of the feature
+- Consider how best to mitigate risk or look for alternate solutions
```

Comment:
**suggestion:** I think it is really important to articulate the problem, or why the changes need to be made. I suggest we add an extra clause to stress the importance of that step.

```
- Clearly articulate the problem and why the changes need to be made
```